{Release} Upgrade to Azure CLI 2.61.0 (#28977)
Azure/azure-cli: src/azure-cli-core/HISTORY.rst

Release History
===============

2.61.0
++++++
* Resolve CVE-2024-34064 (#28895)

> * [BREAKING CHANGE] `az login`: Use WAM as the default authentication method on Windows. If you encounter any issue and want to revert to the previous browser-based authentication method, run `az account clear`, `az config set core.enable_broker_on_windows=false` and `az login` (#28841)

Reviewer: ```suggestion
* [BREAKING CHANGE] `az login`: Use WAM as the default authentication method on Windows. For more details, see https://go.microsoft.com/fwlink/?linkid=2271136 (#28841)
```
